stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequestV2.kt

```diff
@@ -151,5 +162,7 @@ class AnalyticsRequestV2(
         internal const val PARAM_CREATED = "created"
         internal const val PARAM_EVENT_NAME = "event_name"
         internal const val PARAM_EVENT_ID = "event_id"
+
+        private const val TAB = "   "
```

Comment:
why is this 3 space

Reply:
hmm nice catch, I think I set the wrong tab size in my IDE
Renamed it to `INDENTATION`